src/openrct2/rct2/S6Exporter.cpp

```diff
@@ -168,8 +168,8 @@ void S6Exporter::Export()
         auto temp = utf8_to_rct2(gS6Info.details);
         safe_strcpy(_s6.info.details, temp.data(), sizeof(_s6.info.details));
     }
-    uint32_t researchedTrackPiecesA[128];
-    uint32_t researchedTrackPiecesB[128];
+    uint32_t researchedTrackPiecesA[128] = { 0 };
```

Comment:
Do you want zero initialize the whole array or just the first member here?

Reply:
The whole array, and it does so: https://godbolt.org/z/Q9o4RE

Reply:
This is the C way of doing it, its also fine for C++ but you want to default initialize via {}

Reply:
Done